src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -221,3 +221,13 @@ The span finishes once the operation has been executed. The span `status` is set
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and `SentryEvent` will be linked when viewing it on the **Issue Details** page in [sentry.io](https://sentry.io).
```

Comment:
```suggestion
When the operation throws an `Exception`, Sentry's SDK associates it with the running span. If you haven't set the SDK to swallow and capture the exception, the span and `SentryEvent` will be shown as linked on the **Issue Details** page in [sentry.io](https://sentry.io).
```

Reply:
Same here, this text is in multiple places and has not been changed.